pbsmetrics/go_metrics.go

```diff
@@ -375,19 +389,26 @@ func (me *Metrics) RecordAdapterPanic(labels AdapterLabels) {
 // RecordAdapterRequest implements a part of the MetricsEngine interface
 func (me *Metrics) RecordAdapterRequest(labels AdapterLabels) {
 	am, ok := me.AdapterMetrics[labels.Adapter]
+	var aam *AdapterMetrics
 	if !ok {
 		glog.Errorf("Trying to run adapter metrics on %s: adapter metrics not found", string(labels.Adapter))
 		return
 	}
 
-	aam := me.getAccountMetrics(labels.PubID).adapterMetrics[labels.Adapter]
+	if me.accountMetrics != nil {
```

Comment:
`me.accountMetrics` should always be non-nil. We should be creating new account metrics the first time we see an account.

Reply:
Corrected